### DISCUSSION TITLE
Help me to use sonnenglas/amazon-mws-laravel please

### DISCUSSION THREAD

**Discussion by PietroLaGrotta** [sonnenglas/amazon-mws-laravel#58] [General]
hi, I'm Pietro, from Bari, Italy.

I'm developing a web app that manage an invetory of products, and push them on Amazon Marketplace. App is Laravel 7 based on and I installed sonnenglas/amazon-mws-laravel to access to Amazon API. 

I'm having trouble getting the system up and running. Can you help me to use library? Have you available documentation, examples or more that i can view?

Thank you in advance.

**Comment by gaurav-webs** [comment]
@PietroLaGrotta 

I would suggest you switch to [SP-API of the amazon](https://github.com/amzn/selling-partner-api-docs) because as per this announcement https://github.com/amzn/selling-partner-api-docs/discussions/1038 of amazon the MWS is deprecating soon.

**Reply by PietroLaGrotta** [reply]
tnx @gaurav-webs. 
I readed on docs that Amazon will publish "SP-API of the amazon"  by the end of September. is it correct information?
